### PR TITLE
USer Location API

### PR DESCRIPTION
**Pull request by KetanShrikhande**
Document update

### CODE REVIEW

File: src/api-reference/locate/userLocations.markdown [hunk 1]
```diff
@@ -32,7 +32,7 @@ This API supports POST only.
 
 1. [Obtain a client ID, secret and sandbox environment](/manage-apps/partner-applications.html) if you don't already have one.
 2. Obtain your [Source partner](#sourcePartner) information. This will be provided along with your application credentials.
-3. This API supports all Valid JWTs whether they be company level, password grants or client_credentials grant. Read the [Getting Started](/api-reference/authentication/getting-started.html) section of [Authentication API](/api-reference/authentication/apidoc.html) for details. Your sandbox will be configured to accept posts from your application.
+3. This API supports all Valid JWTs specifically client_credentials grant. Read the [Getting Started](/api-reference/authentication/getting-started.html) section of [Authentication API](/api-reference/authentication/apidoc.html) for details. Your sandbox will be configured to accept posts from your application.
```
Comment (charlieconcur) [inline src/api-reference/locate/userLocations.markdown]: More properly stated:

This API supports all valid authorization tokens specifically those with the `client_credentials` grant.

File: src/api-reference/locate/userLocations.markdown [hunk 2]
```diff
@@ -177,14 +177,20 @@ error codes.
...
 Country Code|Mobile Number
 ---|---
 Empty|7160986233
-US|2125553423
-ZA|716098992
+JP|800122334
+81|800122334
 
-### Request
+* When mobileCountryCode is non-blank it will validate the mobile number against that country’s market. It will accept the country letters (in this case JP) or the country code (in this case 81)
```
Comment (charlieconcur) [inline src/api-reference/locate/userLocations.markdown]: Be sure to tick the key names:

When `mobileCountryCode` is non-blank...

**Review by charlieconcur — CHANGES_REQUESTED** [review]
Good to go from my perspective with the requested changes.

**Comment by KetanShrikhande** [comment]
> Good to go from my perspective with the requested changes.

Thanks Charlie - I have updated the documentation per the review comments.

**Review by charlieconcur — APPROVED** [review]

**Pull request merged**